src/integrations/Lemnisk/browser.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable */
+import Logger from "../../utils/logger";
+import { LOAD_ORIGIN } from '../ScriptLoader';
+import { NAME } from './constants';
+
+const logger = new Logger(NAME);
+class LemniskMarketingAutomation {
+  constructor(config, analytics) {
+    if (analytics.logLevel) {
+      logger.setLogLevel(analytics.logLevel);
+    }
+    this.accountId = config.accountId;
+    this.sdkWriteKey = config.sdkWriteKey;
+    this.name = NAME;
+    this._ready = false;
+
+  }
+
+  init() {
+    logger.debug('===in init Lemnisk Marketing Automation===');
+    (function (window, tag, o, a, r) {
+      window.lmSMTObj = window.lmSMTObj || [];
+      var methods = [
+        "init",
+        "page",
+        "track",
+        "identify"
+      ];
+      for (var i = 0; i < methods.length; i++) {
+        lmSMTObj[methods[i]] = function (methodName) {
+          return function () {
+            lmSMTObj.push([methodName].concat(Array.prototype.slice.call(arguments)));
+          };
+        }(methods[i]);
+      }
+      a = o.getElementsByTagName('head')[0];
+      r = o.createElement('script');
+      r.setAttribute('data-loader', LOAD_ORIGIN);
+      r.type = "text/javascript";
+      r.async = 1;
+      r.src = tag;
+      a.appendChild(r);
+    })(
+      window,
+      document.location.protocol === 'https:' ? 'https://cdn25.lemnisk.co/ssp/st/' + this.accountId + '.js' : 'http://cdn25.lemnisk.co/ssp/st/' + this.accountId + '.js',
+      document
+    );
+    window.lmSMTObj.init(this.sdkWriteKey);
+    this._ready = true;
+  }
+
+  isLoaded() {
+    logger.debug("===In isLoaded Lemnisk Marketing Automation===");
+    return !!window.lmSMTObj;
+  }
+
+  isReady() {
+    logger.debug('===In isReady Lemnisk Marketing Automation===');
+    return !!window.lmSMTObj;
+  }
+
+  identify(rudderElement) {
+    logger.debug('===In Lemnisk Marketing Automation identify===');
+    const userId = rudderElement.message.userId || rudderElement.message.anonymousId;
+    if (!userId) {
+      logger.debug('[LemniskMarketingAutomation] identify:: user id is required');
```

Comment:
We should have an `logger.error` here so that user can identify the issue via browser console when the sdk is loaded on their website

Reply:
Done